test/provisioner/provision_azure.properties

```diff
@@ -12,3 +12,5 @@ AZURE_IMAGE_ID=""
 SSH_USERNAME=""
 AZURE_CLI_AUTH="false"
 IS_CI_MANAGED_CLUSTER="false"
+KBS_IMAGE="quay.io/karikjoshi21/kbs/coco-as-21705eb"
```

Comment:
Is this intentional ?

Reply:
Yes, we will merge this PR after https://github.com/confidential-containers/kbs/pull/170 so once this is merged we will have some official image to use.